### ISSUE TITLE
All non-GET requests are failing 

### ISSUE THREAD

**Issue by sibyl229** [WormBase/names#37]
@mgrbyte you might want to checkout the latest fix for the misbehaving redirect on develop branch before proceeding.

On the develop branch, all the non-GET requests (create/update/kill) are failing. The GET requests (search and get a gene) are working for me. I wonder if this might have anything do do with #36 . Is there anything on the UI side that needs to be done differently as a result of #36 ?

The failed response is:
```
{"type":"unknown-exception","class":"datomic.impl.Exceptions$IllegalArgumentExceptionInfo"}
```

**Comment by sibyl229** [comment]
I restored feature/auth-redux branch from #36 to verify that the same failure occurs there.

**Comment by mgrbyte** [comment]
This due to the absence of new schema (in the db) introduced by #36 
Quick fix is:
```
(def new-schema [ #:db{:ident :person/auth-token
      :valueType :db.type/string
      :cardinality :db.cardinality/one
      :unique :db.unique/identity}])
(require '[wormbase.db :as wdb])
(require '[datomic.api :as d])
@(d/transact wdb/conn new-schema)
```` 
Long term fix will come with #7
